src/main/java/cow/commands/DeleteCommand.java

```diff
@@ -0,0 +1,36 @@
+package cow.commands;
+
+import cow.exceptions.CowExceptions;
+import cow.filesaver.FileSaver;
+import cow.message.Ui;
+import cow.tasks.Task;
+import cow.todolist.TodoList;
+
+// solution below inspired by https://github.com/se-edu/addressbook-level2/tree/master
+
+/** Creates a delete command object. **/
+public class DeleteCommand extends Command {
+    public static final String COMMAND_WORD = "delete";
```

Comment:
Great that you adhered to the Java coding standard of using screaming snake case for constants, and using a common prefix for associated constants